src/Sparrow.Server/ByteString.cs

```diff
@@ -446,7 +446,7 @@ public ReadOnlySpan<byte> ToReadOnlySpan()
         }
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public Span<byte> ToSpan()
+        public readonly Span<byte> ToSpan()
```

Comment:
It looks the fix for this warning was reverted with other changes

Reply:
Fixed